Common/Source/Draw/LKProcess.cpp

```diff
@@ -870,6 +870,15 @@ bool MapWindow::LKFormatValue(const short lkindex, const bool lktitle, TCHAR *Bu
 			_stprintf(BufferUnit, TEXT("%s"),(Units::GetHorizontalSpeedName()));
 			_stprintf(BufferTitle, TEXT("%s"), Data_Options[lkindex].Title );
 			break;
+
```

Comment:
Done, please remember next time to create a separate branch, in order to avoid pushing your master. It can be bad sometimes.